api/src/foreign.rs

```diff
@@ -67,6 +68,8 @@ where
 	pub doctest_mode: bool,
 	/// foreign check middleware
 	middleware: Option<ForeignCheckMiddleware>,
+	/// Stored keychain mask (in case the stored wallet seed is tokenized)
+	keychain_mask: Option<SecretKey>,
```

Comment:
This is now just stored by a Foreign API instance, for the case where you want to run a foreign and owner API in the same process but only reference a single instance of the wallet.